cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,85 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to make use of a Command in your application
+================================================
+
+The Console page of the Components section (:doc:`/components/console/introduction`) covers
+how to create a console command. This cookbook article covers how to use a console command
+directly in your application.
+
+Execute Command directly from Controller
+----------------------------------------
+
+Symfony allows you to directly execute a ``Command`` inside your Controller::
+
+    // src/AcmeBundle/Controller/GreetController.php
```

Comment:
In modern Symfony documentation we no longer use `Acme` or "fake looking" examples (`hello`, `demo`, etc.) So this example could be refactored as follows_

```php
// src/AppBundle/Controller/GreetController.php
namespace AppBundle\Controller;

use Symfony\Component\Console\Output\StreamOutput;
use Symfony\Component\Console\Input\ArrayInput;
use Symfony\Bundle\FrameworkBundle\Console\Application;

class GreetController
{
    public function greetAction($name)
    {
        $kernel = $this->get('kernel');
        $application = new Application($kernel);
        $application->setAutoExit(false);

        $input = new ArrayInput(array('command' => 'app:greet'));
        $output = new StreamOutput(tmpfile(), StreamOutput::VERBOSITY_NORMAL);
        $application->run($input, $output);

        rewind($output->getStream());
        $content = stream_get_contents($output->getStream());
        fclose($output->getStream());

        return $content;
    }
}
```

I know it's a lot of work, so please **don't change anything for now** and let's wait for the opinion of the documentation reviewers (@WouterJ and @xabbuh).